R/globals.R

```diff
@@ -8,18 +8,18 @@ utils::globalVariables(c(
   "week", # <outbreak_model>
   "onset", # <outbreak_model>
   "cumulative", # <outbreak_model>
-  "asym", # <outbreak_setup>
+  "asymptomatic", # <outbreak_setup>
```

Comment:
duplicated in line 16?

Reply:
This is the default behaviour of {roxyglobals}, see https://github.com/anthonynorth/roxyglobals/issues/6. If you'd prefer each global variable to only be listed once let me know and I can make the change in a separate PR.